examples/dcgan_fashiongen/create_mar.sh

```diff
@@ -16,7 +16,6 @@ function cleanup {
 trap cleanup EXIT
 
 # Download and Extract model's source code
-sudo apt-get install zip unzip -y
```

Comment:
is this some other PR?

Reply:
Initially, I ended up fixing one of the failing tests to make it work in docker. Thought I'll keep it. Should I create a separate PR for this?

Reply:
Na it's fine